src/main/flight/position.h

```diff
@@ -20,22 +20,19 @@
 
 #pragma once
 
-#include "common/time.h"
+#include "pg/position.h"
 
 #define TASK_ALTITUDE_RATE_HZ 100
 
-typedef struct positionConfig_s {
-    uint8_t altitude_source;
-    uint8_t altitude_prefer_baro;
-    uint16_t altitude_lpf;                // lowpass cutoff (value / 100) Hz for altitude smoothing
-    uint16_t altitude_d_lpf;              // lowpass for (value / 100) Hz for altitude derivative smoothing
-} positionConfig_t;
+typedef enum {
+    ALTITUDE_SOURCE_DEFAULT = 0,
```

Comment:
Maybe ALTITUDE_SOURCE_AUTO ?

Also, what if rangefinder is added? It will get complicated quickly ...